packages/cicero-core/src/template.js

```diff
@@ -172,14 +179,89 @@ class Template {
         return hasher.digest('hex');
     }
 
+    /**
+     * verifies the signature stored in the template object using the template hash and timestamp
+     * @return {boolean} true if signature is valid else false
```

Comment:
If the signature is invalid, this method throws an error. I would instead remove the `@return` parameter altogether.